src/RenderWebGL.js

```diff
@@ -223,6 +223,30 @@ class RenderWebGL extends EventEmitter {
     }
 
     /**
+     * Update an existing SVG skin, or create an SVG skin if the previous skin was not SVG.
+     * @param {!int} skinId the ID for the skin to change.
+     * @param {!string} svgData - new SVG to use.
+     * @param {?Array<number>} rotationCenter Optional: rotation center of the skin. If not supplied, the center of the
+     * skin will be used
+     * @returns {!int} the ID for the skin. This will be different from skinId if the old skin was not SVG.
+     */
+    updateSVGSkin (skinId, svgData, rotationCenter) {
+        if (this._allSkins[skinId] instanceof SVGSkin) {
+            this._allSkins[skinId].setSVG(svgData, rotationCenter);
+            return skinId;
+        }
+
+        const newSkinId = this.createSVGSkin(svgData, rotationCenter);
+        for (const drawable of this._allDrawables) {
+            if (drawable && drawable.skin.id === skinId) {
+                drawable.skin = this._allSkins[newSkinId];
+            }
+        }
+        this.destroySkin(skinId);
+        return newSkinId;
```

Comment:
Instead of this `for` loop, what do you think of something like this?
```js
const newSkin = new SVGSkin(skinId, this);
newSkin.setSVG(svgData, rotationCenter);
const oldSkin = this._allSkins[skinId];
this._allSkins[skinId] = newSkin;
delete oldSkin;
```

Reply:
That seems better, then I don't have to worry about propagating the new skin ID in the VM.

Did you mean oldSkin.dispose() instead of delete oldSkin?

Reply:
Oh, wow... yep, I meant `oldSkin.dispose()`. I guess I had a C++ flashback there...